### PR TITLE
chore: add github workflows to release package

### PR DESCRIPTION
**Pull request by RadxaPanda**

### CODE REVIEW

File: .github/workflows/new_version.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          sudo apt-get install --no-install-recommends -y git-buildpackage
+          export DEBEMAIL="dev@radxa.com"
+          export DEBFULLNAME='"Radxa Computer Co., Ltd"'
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          EDITOR=true gbp dch --debian-branch=main --release --dch-opt=--upstream
+      - name: Test
+        run: |
+          sudo apt-get install debhelper devscripts dh-exec lintian meson ninja-build libglib2.0-dev-bin appstream-util
+          debuild --no-lintian --lintian-hook "lintian --fail-on error,warning --suppress-tags bad-distribution-in-changes-file -- %p_%v_*.changes" --no-sign -b
```
Comment (RadxaYuntian) [inline .github/workflows/new_version.yml]: Use similar Makefile in other repo as the build entry

File: .github/workflows/new_version.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          sudo apt-get update
+          sudo apt-get install --no-install-recommends -y git-buildpackage
+          export DEBEMAIL="dev@radxa.com"
+          export DEBFULLNAME='"Radxa Computer Co., Ltd"'
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          EDITOR=true gbp dch --debian-branch=main --release --dch-opt=--upstream
+      - name: Test
+        run: |
+          sudo apt-get install debhelper devscripts dh-exec lintian meson ninja-build libglib2.0-dev-bin appstream-util
```
Comment (RadxaYuntian) [inline .github/workflows/new_version.yml]: They should be `Build-Deps` in `debian/control` file.

File: .github/workflows/new_version.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          debuild --no-lintian --lintian-hook "lintian --fail-on error,warning --suppress-tags bad-distribution-in-changes-file -- %p_%v_*.changes" --no-sign -b
+      - name: Push
+        run: |
+          git push
```
Comment (RadxaYuntian) [inline .github/workflows/new_version.yml]:
```suggestion
          git push

```

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (RadxaYuntian) [inline .github/dependabot.yml]:
```suggestion
      interval: "daily"

```

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+dch: debian/changelog
+	EDITOR=true gbp dch --debian-branch=main --release --dch-opt=--upstream
+
+.PHONY: deb
+deb: debian
+	debuild --no-lintian --lintian-hook "lintian --fail-on error,warning --suppress-tags bad-distribution-in-changes-file -- %p_%v_*.changes" --no-sign -b
+
+.PHONY: release
+release:
+	gh workflow run .github/workflows/new_version.yml
```
Comment (RadxaYuntian) [inline Makefile]: Please check `rsetup` for the latest Release section.

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

File: Makefile [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+dch: debian/changelog
+	EDITOR=true gbp dch --debian-branch=main --release --dch-opt=--upstream
+
+.PHONY: deb
+deb: debian
+	debuild --no-lintian --lintian-hook "lintian --fail-on error,warning --suppress-tags bad-distribution-in-changes-file -- %p_%v_*.changes" --no-sign -b
+
+.PHONY: release
+release:
+	gh workflow run .github/workflows/new_version.yml --ref $(shell git branch --show-current)
```
Comment (RadxaYuntian) [inline Makefile]:
```suggestion
	gh workflow run .github/workflows/new_version.yml --ref $(shell git branch --show-current)

```

File: README.md [hunk 7]
```diff
@@ -11,8 +11,7 @@ python3 ./launch.py
 To package for debian, run the following command:
 
 ```
-debuild --no-lintian --lintian-hook "lintian --fail-on error,wa
-rning --suppress-tags bad-distribution-in-changes-file -- %p_%v_*.changes" --no-sign -b
+debuild --no-lintian --lintian-hook "lintian --fail-on error,warning --suppress-tags bad-distribution-in-changes-file -- %p_%v_*.changes" --no-sign -b
```
Comment (RadxaYuntian) [inline README.md]:
```suggestion
make deb
```

File: meson.build [hunk 8]
```diff
@@ -1,6 +1,6 @@
 project('welcome',
-          version: '0.1.0',
-    meson_version: '>= 0.62.0',
+          version: '0.0.2',
+    meson_version: '>= 0.61.0',
```
Comment (RadxaYuntian) [inline meson.build:3]: meson_version becomes smaller?

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

File: meson.build (hunk 8 above)
Comment (RadxaPanda) [reply meson.build:3]: yes 0.62 isnt available on non sid releases and i think ubuntu

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
squash

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**